packages/terra-spacer/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-spacer",
+  "main": "lib/Spacer.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-spacer",
+    "Spacer",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.6.1"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-bps": "0.0.4",
+    "terra-base": "^1.0.0",
```

Comment:
Update this to use the same version in your peer dependencies.

Reply:
Updated https://github.com/cerner/terra-core/commit/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-f3bfa5e0bf12fb4af2128aaf883cc625R31